amundsen_application/static/css/_buttons-default.scss

```diff
@@ -73,10 +73,6 @@
     box-shadow: 0 1px 3px 0 rgba(17, 17, 31, 0.12);
   }
```

Comment:
You can actually delete both `.btn-shadow` classes. I added them in because they were in one of Weibke's mocks, and then removed them since they weren't part of the standard buttons that Matt produced.